update locale select to have all languages that TM supports

willemarcel opened the pull request.

xamanu commented on frontend/src/index.js:
```diff
@@ -3,13 +3,34 @@ import ReactDOM from 'react-dom';
 import WebFont from 'webfontloader';
 import { addLocaleData } from 'react-intl';
-import de from 'react-intl/locale-data/de';
+
```
Oh wow, we won't have all the languages at the beginning. Planned are Spanish, French, Turkish, Filipino, Indonesian, Swahili to start with.

willemarcel commented:
@xamanu I just added support to Swahili, Turkish and Filipino (Tagalog) that were missing on TM. Furthermore, I disabled the selection of the languages that we are not supporting in the first release and fixed a bug in the map localization when the locale code is bigger than 2 chars.

willemarcel commented:
@xamanu I renamed "Tagalog" to "Filipino"
